containers/nginx-storage/delete-old-files.sh

```diff
@@ -28,34 +37,39 @@ function delete_empty_directories {
     done
 }
 
+function check_free_space {
+    local free_space="$(df | grep '/srv/storage' | awk '{ print $4; }')"
+    if [ -z "$free_space" ]; then
+        echo "[crond][error][disk cleanup]: There is a problem with checking free space on disk. Please check if mount point \"/srv/storage/\" exist."
+        exit 1
+    fi
+    # Convert kilobyte to gigabyte
+    let free_space=$free_space/1048576
+    echo $free_space
+}
+
 for directory in ${DIRECTORIES[@]}; do
     if [ ! -d "$directory" ]; then
         echo "[crond][error][disk cleanup]: Please check if the \"$directory\" directory exist."
         exit 1
     fi
 done
 
-free_space="$(df | grep '/srv/storage' | awk '{ print $4; }')"
-if [ -z "$free_space" ]; then
-    echo "[crond][error][disk cleanup]: There is a problem with checking free space on disk. Please check if mount point \"/srv/storage/\" exist."
-    exit 1
-fi
-
-delete_old_files 30
+available_free_space=$(check_free_space)
+delete_old_files $MAXIMUM_DAYS
 
-# Convert kilobyte to gigabyte
-let free_space=$free_space/1048576
-days=29
-while (( 15 > $free_space ))
+let MAXIMUM_DAYS--
```

Comment:
I would add third variable: 
```
let CURRENT_NUMBER_OF_DAYS_TO_DELETE=$MAXIMUM_DAYS-1
```
or sth similar (even `DAYS_TO_DELETE`)
and then use it in calculations, not to modify `MAXIMUM_DAYS`

Reply:
Done.